### PR TITLE
If bookmark is enabled added padding-right so text does not cover bookmark icon

### PR DESCRIPTION
**Pull request by MaksymShokin**
@tonytlwu @squallstar 

## Issue
Fliplet/fliplet-studio#4913

## Description
If bookmark if enabled class bookmark-padding will be added to div with all text.

## Screenshots/screencasts
Bookmark demo

## Backward compatibility
This change is fully backward compatible.

### CODE REVIEW

File: templates/build/small-card-loop.build.hbs [hunk 1]
```diff
@@ -10,7 +10,7 @@
         \{{/if}}
       </div>
     </div>
-    <div class="small-card-list-text-wrapper">
+    <div class="small-card-list-text-wrapper \{{#if bookmarksEnabled}} bookmark-padding \{{/if}}">
```
Comment (squallstar) [inline templates/build/small-card-loop.build.hbs]: I would recommend calling this `has-bookmarks`, @tonytlwu do you agree?

Comment (squallstar) [reply templates/build/small-card-loop.build.hbs]: @tonytlwu ^

Comment (tonytlwu) [reply templates/build/small-card-loop.build.hbs]: I agree

Comment (MaksymShokin) [reply templates/build/small-card-loop.build.hbs]: @tonytlwu @squallstar Just to confirm in this case I will have to change the name from bookmarksEnabled to has-bookmarks though all the code. 
bookmarks demo

Comment (squallstar) [reply templates/build/small-card-loop.build.hbs]: @maksim-an-upplabs nope, what we meant is to change your `bookmark-padding` class to be named `has-bookmarks`.

Comment (MaksymShokin) [reply templates/build/small-card-loop.build.hbs]: @squallstar Fixed

**Pull request merged**